test/e2e/tests/test_subscription_list_endpoints.py

```
@@ -186,6 +193,97 @@ def test_subscription_includes_model_refs(self):
+        subscription_name = "e2e-enrichment-sub"
+        expected_display_name = "E2E Enrichment Test Model"
+        expected_description = "Model created by e2e test to verify display_name/description enrichment"
+
+        try:
+            sa_token = _create_sa_token(sa_name, namespace=sa_ns)
+            sa_user = _sa_to_user(sa_name, namespace=sa_ns)
+
+            # Create a MaaSModelRef with display-name and description annotations.
+            # Points to the existing e2e-distinct-simulated LLMIS so the controller
+            # can reconcile the subscription to Active.
+            _apply_cr({
```

Ruff (0.15.12)

[error] Undefined name `_apply_cr`

(F821)